src/main/resources/application.properties

```diff
@@ -1,11 +1,2 @@
 spring.application.name=book-store
-spring.datasource.url=jdbc:mysql://localhost:3306/booksdb?serverTimeZone=UTC
-spring.datasource.username=root
-spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
-
-spring.jpa.hibernate.ddl-auto=validate
-spring.jpa.show-sql=true
 spring.jpa.open-in-view=false
```

Comment:
Better return it if you want to have the possibility to run your project locally